persistence/src/main/scala/hmda/persistence/processing/SubmissionManager.scala

```diff
@@ -45,6 +46,7 @@ class SubmissionManager(submissionId: SubmissionId) extends HmdaActor {
   val supervisor = context.parent
   val hmdaFilingF = (supervisor ? FindHmdaFiling(period)).mapTo[ActorRef]
 
+  val submissionLarStats: ActorRef = context.actorOf(SubmissionLarStats.props(submissionId), s"submission-lar-stats-${submissionId.toString}")
```

Comment:
It looks like this `val` is never used in this file. If that's the case, what is this line for? Does it ensure that the `SubmissionLarStats` actor gets started?